client/my-sites/upgrades/domain-management/list/domain-only.jsx

```diff
@@ -10,19 +10,27 @@ import React, { PropTypes } from 'react';
 import EmptyContent from 'components/empty-content';
 import { domainManagementEdit } from 'my-sites/upgrades/paths';
 
-const DomainOnly = ( { domainName, siteId, translate } ) => (
-	<EmptyContent
-		title={ translate( '%(domainName)s is ready when you are.', { args: { domainName } } ) }
-		line={ translate( 'Start a site now to unlock everything WordPress.com can offer. Or add email and manage your domain preferences.' ) }
-		action={ translate( 'Create Site' ) }
-		actionURL={ `/start/site-selected/?siteSlug=${ encodeURIComponent( domainName ) }&siteId=${ encodeURIComponent( siteId ) }` }
-		secondaryAction={ translate( 'Manage Domain' ) }
-		secondaryActionURL={ domainManagementEdit( domainName, domainName ) }
-		illustration={ '/calypso/images/drake/drake-browser.svg' } />
+const DomainOnly = ( { domainName, hasNotice, siteId, translate } ) => (
+	<div>
+		<EmptyContent
+			title={ translate( '%(domainName)s is ready when you are.', { args: { domainName } } ) }
+			line={ translate( 'Start a site now to unlock everything WordPress.com can offer. Or add email and manage your domain preferences.' ) }
+			action={ translate( 'Create Site' ) }
+			actionURL={ `/start/site-selected/?siteSlug=${ encodeURIComponent( domainName ) }&siteId=${ encodeURIComponent( siteId ) }` }
+			secondaryAction={ translate( 'Manage Domain' ) }
+			secondaryActionURL={ domainManagementEdit( domainName, domainName ) }
+			illustration={ '/calypso/images/drake/drake-browser.svg' } />
+		{ hasNotice && (
+			<div className="domain-only-site__settings-notice">
+				{ translate( 'New domains usually start working immediately, but may be unreliable for the first few hours.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Your domain should start working immediately, but may be unreliable during the first 72 hours.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).